src/server/app.js

```diff
@@ -1,12 +1,17 @@
+import { Paratii } from 'paratii-lib/dist/paratii'
+import { getParatiiConfig } from 'utils/AppUtils'
```

Comment:
nitpicky, but I would use `require` since the rest of the file does that. to be consistent

Reply:
ok